src/Microsoft.Health.Fhir.TemplateManagement/ArtifactProviders/DefaultTemplateCollectionProvider.cs

```diff
@@ -55,8 +55,9 @@ public DefaultTemplateCollectionProvider(IMemoryCache templateCache, TemplateCol
             var ccdaDefaultTemplatesTask = Task.Run(() => templates.Add(ExtractTemplatesFromResource(DefaultTemplateInfo.CcdaDefaultTemplatesResource, DataType.Ccda.ToString())), cancellationToken);
             var jsonDefaultTemplatesTask = Task.Run(() => templates.Add(ExtractTemplatesFromResource(DefaultTemplateInfo.JsonDefaultTemplatesResource, DataType.Json.ToString())), cancellationToken);
             var stu3ToR4DefaultTemplatesTask = Task.Run(() => templates.Add(ExtractTemplatesFromResource(DefaultTemplateInfo.Stu3ToR4DefaultTemplatesResource, DataType.Fhir.ToString())), cancellationToken);
+            var fhirToHl7v2DefaultTemplatesTask = Task.Run(() => templates.Add(ExtractTemplatesFromResource(DefaultTemplateInfo.FhirToHl7v2DefaultTemplatesResource, "FhirToHl7v2")), cancellationToken);
```

Comment:
Is the new enum not accessible here?

Reply:
it is - thanks - updated to use the enum